app/views/match_contacts/_match_contacts_list.haml

```diff
@@ -9,12 +9,17 @@
       - Translation.translate('Stabilization Service Providers')
       - Translation.translate('Housing Search Providers')
       .c-detail-group__title--secondary= Translation.translate(title)
-      .detail-box--value
-        - match.send(contact_type).each do |contact|
+      .detail-box--value 
+        - contact_type_column = contact_type.to_s.sub('_contacts','').to_sym #TODO: This feels too hacky
```

Comment:
Can we move this into the `ClientOpportunityMatchContact` class?  It could have a method:
```
ClientOpportunityMatchContact.column_for(contact_type)
```

Ideally that would lookup the column in a has keyed on type.